### ISSUE TITLE
Change FontColor of title + axes / hide zero values

### ISSUE THREAD

**Issue by keineAhnung667** [mtatsuma/MMM-WeatherChart#29]
Hello mtatsuma,

Could you please add an option to change the FontColor of the title an the axes? Its gray by default.
I changed it manually with `Chart.defaults.global.defaultFontColor = this.config.color`

And another thing: 
Could you add the ability to hide zero values in the rain chart? So if there is no rain, there is no text with "0"
(Like here: https://stackoverflow.com/questions/58425747/how-to-hide-zero-data-value-from-chartjs)

![grafik](https://user-images.githubusercontent.com/71514327/122448194-cadd0280-cfa4-11eb-8fbe-de8e662c1d4c.png)

**Comment by mtatsuma** [comment]
Hi

Thank you for using the module.

I have dealt with the both requests.
Please clone the module from the latest master branch and try it.

NOTE: the latest master branch has some breaking changes because I bumped up Chart.js major version (v2.x -> v3.x).

**Comment by keineAhnung667** [comment]
Thanks for your effort.
Im a little bit confused because in your Readme its written
`git clone -b v2.6.0 https://github.com/mtatsuma/MMM-WeatherChart.git`

Can i update to the latest version (3.x) or do i have to wait?

**Comment by mtatsuma** [comment]
I was waiting for the chart.js datalabels plugin v2.0.0 to be released and it was released yesterday!
https://github.com/chartjs/chartjs-plugin-datalabels/commit/b04395850bed5729e03e4adb150762819e85898b

Then, I will release v3.0.0 soon. Please give me a moment.

**Comment by mtatsuma** [comment]
I have released v3.0.0.
https://github.com/mtatsuma/MMM-WeatherChart/releases/tag/v3.0.0

To install it, please try
```
git clone -b v3.0.0 https://github.com/mtatsuma/MMM-WeatherChart.git
```

**Comment by keineAhnung667** [comment]
Hi mtatsuma,
I tried Version 3.0 now but there are some mistakes:
![Unbenannt](https://user-images.githubusercontent.com/71514327/126893414-8f8c9a40-0648-4c26-8d28-cd74404e5513.png)

The text is cut and there are temperature labels missing.

My config:

        {
            module: "MMM-WeatherChart",
	    //disabled: true,
            position: "top_right",
            config: {
                apiKey: "97b41407fe2e4fe55643ec47d00c5430",
                lat: 51.961563,
                lon: 7.628202,
                units: "metric",
                updateInterval: 60 * 60 * 1000, //60 min
                title: "Wetter Münster",
		fontSize: 20,
                showRain: true,
		showZeroRain: false,
		includeSnow: true,
                showIcon: false,
		iconWidth: 40,
		iconHeight: 40,
                fillColor: "rgba(30, 144, 255, 0.4)",
                dataNum: 13,
                dataType: "hourly",
		nightBorderDash: [5, 0],
                height: "300px",
                width: "600px",
            }
        },

Is my config wrong?

**Comment by mtatsuma** [comment]
I'm sorry for this late response and thank you for sharing your problem.
I also reproduced the issue and I will fix that.
Please give me some time.

**Comment by mtatsuma** [comment]
I have fixed the issue.
Please clone the latest module
```
$ git clone https://github.com/mtatsuma/MMM-WeatherChart.git
```
and try it.

By the way, iconWidth and iconHeight in your config is no longer available from v3.0.0 because it does not work well for v3.x Chart.js.

**Comment by keineAhnung667** [comment]
Thanks,
works again :)
